config/config_cheyenne_gnu.sh

```diff
@@ -2,34 +2,34 @@
 
 # Compiler/MPI combination
 export HPC_COMPILER="gnu/10.1.0"
-export HPC_MPI="mpt/2.22"
-export HPC_PYTHON="python/dummy"
+export HPC_MPI="mpt/2.25"
+export HPC_PYTHON="python/3.4.6"
 
 # Build options
 export USE_SUDO=N
 export PKGDIR=pkg
 export LOGDIR=log
-export OVERWRITE=Y
-export NTHREADS=4
+export OVERWRITE=N
+export NTHREADS=8
 export   MAKE_CHECK=N
 export MAKE_VERBOSE=N
 export   MAKE_CLEAN=N
 export DOWNLOAD_ONLY=N
 export STACK_EXIT_ON_FAIL=Y
-export WGET="wget -nv"
+export WGET="wget -nv --no-check-certificate"
 
 module purge
-module unuse /glade/u/apps/ch/modulefiles/default/compilers
```

Comment:
Have you tried this with netcdf 4.8.1? My guess is that you will run into trouble, but hopefully I am wrong.

Reply:
Yes, this build was made with netcdf version_c 4.8.1 and version_f 4.5.4